crates/ibc/src/core/ics03_connection/handler/conn_open_confirm.rs

```diff
@@ -39,15 +39,17 @@ pub(crate) fn process(
         conn_end.delay_period(),
     );
 
-    // 2. Pass the details to the verification function.
-    verify_proofs(
+    verify::verify_connection_proof(
         ctx,
-        None,
         msg.proofs.height(),
         &conn_end,
         &expected_conn,
-        &msg.proofs,
+        msg.proofs.height(),
+        msg.proofs.object_proof(),
     )?;
+    if let Some(proof) = msg.proofs.consensus_proof() {
```

Comment:
I don't think this check is required because `MsgConnectionOpenConfirm` will never have an associated consensus proof.

Reply:
Good catch [af02ad3](https://github.com/cosmos/ibc-rs/pull/160/commits/af02ad362d09d8b10234ec414d5a10f11d59d1e0)